packages/common/src/schemas/upload/uploadFormSchema.ts

```diff
@@ -269,7 +269,6 @@ export const createCollectionSchema = (collectionType: 'playlist' | 'album') =>
       }),
       is_private: z.optional(z.boolean()),
       is_album: z.literal(collectionType === 'album'),
-      is_private: z.optional(z.boolean()),
```

Comment:
this is unrelated but figured I'd sneak it in

Reply:
🔪

Reply:
This is actually currently breaking web-init so please do merge this PR whenever you get a chance 👌